src/main/java/frc/robot/Constants.java

```diff
@@ -57,10 +57,10 @@
 
 
     public static class CameraMode {
-        public static final double VISION = 0, DRIVER =1;
+        public static final int VISION = 0, DRIVER =1;
```

Comment:
Space before assignment please.

```suggestion
        public static final int VISION = 0, DRIVER = 1;
```